### ISSUE TITLE
Update engine limitations in package.json

### ISSUE THREAD

**Issue by sigoden** [duartealexf/sql-ddl-to-json-schema#74]
When installing the package with latest node, npm throws
```
error sql-ddl-to-json-schema@4.0.4: The engine "node" is incompatible with this module. Expected version ">=8.6.0 <=14". Got "16.13.0"
```

Could you upgrade node modules and node version?

**Comment by duartealexf** [comment]
Sure, I had this in my 'to do' list for a while, will do it as soon as I get the chance. Thanks.

**Comment by duartealexf** [comment]
@sigoden I just released a fix! Update to `sql-ddl-to-json-schema@4.0.6`.
